sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/data/model/MoleculesItem.kt

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package uk.gov.hmrc.components.compose.molecule.titleBodyView
+package uk.gov.hmrc.sample_compose_fragments.data.model
 
-class TitleBodyView
+import androidx.annotation.StringRes
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.ComponentItem
+
+class MoleculesItem(val id: Int, @StringRes override val titleRes: Int): ComponentItem
```

Comment:
can we use AtomItem model (with a common name eg: Component item model)class in all the tab except colours because they uses the common params?